client/src/settings/mdn-worker.tsx

```diff
@@ -84,13 +84,13 @@ export class MDNWorker {
   }
 
   async enableServiceWorker(type: SwType) {
-    if ("serviceWorker" in navigator && !this.registered) {
-      await navigator.serviceWorker.register(this.swName(type), {
-        scope: "/",
-      });
-      this.registered = true;
-    }
-    this.registerMessageHandler();
+    // if ("serviceWorker" in navigator && !this.registered) {
```

Comment:
I was debating it - at least locally, the service worker was messing with api requests, but this might be a quirk of the local proxy stuff. We'll want to remove the api intercept in the service worker before deploying, but I'll uncomment this for now.